.github/actions/build-release/action.yml

```diff
@@ -0,0 +1,133 @@
+name: "Build Gleam"
+description: "Build Gleam Release"
+inputs:
+  version:
+    description: "Build Version"
+    required: true
+  toolchain:
+    description: "Cargo Toolchain"
+    required: true
+  target:
+    description: "Cargo Installation Target"
+    required: true
+  cargo-tool:
+    description: "Cargo Tool" # TODO: What is this? describe...
+    required: true
+  binary:
+    description: "Binary" # TODO: What is this? describe...
```

Comment:
TODO: Put a better description

Reply:
This is the expected binary architecture, used to verify the binary has been correctly produced. `expected-binary-architecture` or such would be a clearer name